pkgs/development/libraries/linenoise/default.nix

```diff
@@ -0,0 +1,30 @@
+  { stdenv, fetchFromGitHub, lib }:
```

Comment:
I see you're using both `stdenv.lib` and `lib`, perhaps stick to the former for consistency (they should be the same thing).